master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,408 @@
+   which will allow all traffic to and from the host endpoints.
+
+   To do this, create a file named `default-host-policy.yaml` with the following contents.
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkPolicy
+   metadata:
+     name: default-host
+   spec:
+     # Select all {{site.prodname}} nodes.
+     selector: running-calico == "true"
+     order: 5000
+     ingress:
+     - action: Allow
+     egress:
+     - action: Allow
+   ```
+
+   Then, use calicoctl to apply this policy.
+
+   ```shell
+   calicoctl apply -f default-host-policy.yaml
+   ```
+
+1. List the nodes on which {{site.prodname}} is running. In this case, we have two nodes in the cluster.
+
+   ```shell
+   $ calicoctl get nodes
+   NAME
+   kubeadm-master
+   kubeadm-node-0
+   ```
+
+1. Create host endpoints for each {{site.prodname}} node.
+
+   Create a file named `host-endpoints.yaml` containing a host endpoint for each node listed
+   above. In this example, the contents would look like this.
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: HostEndpoint
+   metadata:
+     name: kubeadm-master.eth0
+     labels:
+       running-calico: "true"
+   spec:
+     interfaceName: eth0
+     node: kubeadm-master
+   ---
+   apiVersion: projectcalico.org/v3
+   kind: HostEndpoint
+   metadata:
+     name: kubeadm-node-0.eth0
+     labels:
+       running-calico: "true"
+   spec:
+     interfaceName: eth0
+     node: kubeadm-node-0
+   ```
+
+   Note the use of a label to indicate that this host endpoint is running {{site.prodname}}. The
+   label matches the selector of the network policy created in step 1.
+
+   Then, use calicoctl to apply the host endpoints with the following command.
+
+   ```shell
+   calicoctl apply -f host-endpoints.yaml
+   ```
+
+1. Create a network policy which restricts access to the {{site.nodecontainer}} prometheus metrics port.
+
+   Now let's create a network policy which limits access to the prometheus metrics port such that
+   only endpoints with the label `calico-prometheus-access: true` can access the metrics.
+
+   To do this, create a file named `calico-prometheus-policy.yaml` with the following contents.
+
+   ```yaml
+   # Allow traffic to prometheus only from sources which are
+   # labeled as such, but don't impact any other traffic.
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkPolicy
+   metadata:
+     name: restrict-calico-node-prometheus
+   spec:
+     # Select all {{site.prodname}} nodes.
+     selector: running-calico == "true"
+     order: 500
+     types:
+     - Ingress
+     ingress:
+     # Deny anything that tries to access the prometheus port
+     # but that doesn't match the necessary selector.
+     - action: Deny
+       protocol: TCP
+       source:
+         notSelector: calico-prometheus-access == "true"
+       destination:
+         ports:
+         - 9091
+   ```
+
+   Then, use calicoctl to apply this policy.
```

Comment:
I think it'd help to spell out what this policy means.  Everyone has trouble with notSelector when they first see it.  might be a good opportunity to educate